lib/fs.js

```diff
@@ -6,6 +6,7 @@
 const constants = process.binding('constants').fs;
 const util = require('util');
 const pathModule = require('path');
+const internalFs = require('internal/fs');
```

Comment:
@rvagg ... I believe the suggestion is to do:

```js
const internalFs = require('internal/fs');
const nullCheck = internalFs.nullCheck;
```

Which seems reasonable to me :-)